src/main/java/me/botsko/prism/actions/BlockAction.java

```diff
@@ -410,15 +411,15 @@ protected ChangeResult placeBlock(Player player, QueryParameters parameters, boo
                 final Skull skull = (Skull) block.getState();
                 skull.setRotation( s.getRotation() );
                 skull.setSkullType( s.getSkullType() );
-                if( s.owner != null && !s.owner.isEmpty() ) {
+                if( !Strings.isNullOrEmpty(s.owner) ) {
```

Comment:
But why? `Strings#isNullOrEmpty` does literally the same thing as the code you replaced